README.md

```diff
@@ -1,2 +1,151 @@
+  - [📝 License ](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog <a name="about-project"></a>
+
+**Blog app** is a classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.. Everything will be based on the [ERD](./erd_1.png).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Built methods using Ruby**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- 
+## 🚀 Live Demo <a name="live-demo"></a>
+[Video presentation](https://drive.google.com/file/d/1fPyQgJ-ofpZteSwd57WQIpwpwEXPJy18/view?usp=sharing) -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+    ruby
+    bundler
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+  git clone https://github.com/khanh-devos/rail-blog-app.git 
+```
+
+### Install
+
+Install this project with:
+
+```bash
+  cd the_new_folder_name
+  bundle install
+```
+
+it will install the required gemfile for running the project
+
+### Usage
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+👤 **Khanh**
+
+- GitHub: [khanh-devos](https://github.com/khanh-devos)
+- Twitter: [@khanhror](https://twitter.com/home?lang=en)
+- LinkedIn: [Khanh RoR](https://www.linkedin.com/in/khanh-dom/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add Database to store recent message**
+- **Add Ui**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/khanh-devos/rail-blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can share this project to your friend
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and all the contributors for this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The link to your `LICENCE` file is broken as you can see below because of a typo in the link: you have `./LICENSE` instead of `./LICENCE`. Please fix this 🙏 
![image](https://github.com/khanh-devos/rail-blog-app/assets/46839548/aa307e89-884e-4785-8d36-823937faa0c9)